fix_simulation/Readme.md

```diff
@@ -0,0 +1,55 @@
+# Trade Simulation
+This project contains a simulated trading application that interacts with a FIX (Financial Information eXchange) server. It simulates placing, modifying, and canceling orders for two clients.
+
+## Prerequisites
+- Docker and Docker Compose installed on your system
+- Python 3.7 or higher
+
+## Setup
+1. Clone this repository to your local machine.
+
+2. Create a `.env` file in the root directory of the project with the following content:
+
+```
+TRUEX_USER1_MNEMONIC=your_user1_mnemonic
+TRUEX_USER1_KEY_ID=your_user1_key_id
+TRUEX_USER1_KEY_SECRET=your_user1_key_secret
+TRUEX_USER2_MNEMONIC=your_user2_mnemonic
+TRUEX_USER2_KEY_ID=your_user2_key_id
+TRUEX_USER2_KEY_SECRET=your_user2_key_secret
```

Comment:
This has probably changed since the PR was created, but we can create a single API key that works for multiple users. We can also use a script that automatically retrieves user IDs

Reply:
Addressed in https://github.com/true-markets/tools/pull/1/commits/81c0e300d2614e848cbb5a185fe4afd1ca9023fc  
Had to programmatically set the session, `SENDER_COMP_ID` as defining these in the `cient.cfg`, quickfix automatically creates FIX sessions and it wasnt trivial to change the `SENDER_COMP_ID`.
There was a workaround which involved having a map of sessionID to client mnemonic, tradeoff was it adds some complexity for the trade simulations.